woocommerce/woocommerce-gutenberg-products-block
# Cart item validation and notices in cart/checkout

Adds validation of cart items to the API and then displays any returned errors in the cart and checkout.
Fixes #2068

### Screenshots

![Screenshot 2020-03-31 at 14 37 34](https://user-images.githubusercontent.com/90977/78033300-200c0480-735e-11ea-8827-0db798e1af90.png)
![Screenshot 2020-03-31 at 14 37 20](https://user-images.githubusercontent.com/90977/78033308-20a49b00-735e-11ea-8d45-5fba3da8de04.png)

### How to test the changes in this Pull Request:

1. Add items to cart, then edit them to either make out of stock, or so there isn't enough stock qty available.
2. View cart - see notice
3. Go to checkout - see notice
4. Request GET `wc/store/cart` and see `errors` array in response.
